aptos-move/framework/aptos-framework/sources/resource_account.move

```diff
@@ -169,6 +171,7 @@ module aptos_framework::resource_account {
         let resource_addr = signer::address_of(resource);
         let (resource_signer_cap, empty_container) = {
             let container = borrow_global_mut<Container>(source_addr);
+            assert!(simple_map::contains_key(&container.store, &resource_addr), error::invalid_argument(EINVALID_SOURCE_ACCOUNT_AND_RESOURCE_ACCOUNT_COMBINATION));
```

Comment:
Why not just EUNAUTHORIZED_NOT_OWNER or something along that line as this error can mean the source  account no longer has control (SignerCap) over the resource account? EINVALID_SOURCE_ACCOUNT_AND_RESOURCE_ACCOUNT_COMBINATION sounds very cryptic

Reply:
good point! updated : )